Controller/Checkout/PlaceOrder.php

```diff
@@ -0,0 +1,81 @@
+<?php
+/**
+ * Copyright © Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * A copy of the License is located at
+ *
+ *  http://aws.amazon.com/apache2.0
+ *
+ * or in the "license" file accompanying this file. This file is distributed
+ * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+namespace Amazon\Pay\Controller\Checkout;
+
+use Magento\Framework\App\ObjectManager;
+use Amazon\Pay\Logger\ExceptionLogger;
+use Magento\Framework\Controller\Result\JsonFactory;
+
+class PlaceOrder extends \Magento\Framework\App\Action\Action
```

Comment:
\Magento\Framework\App\Action\Action is deprecated.
We should use either \Magento\Framework\App\ActionInterface or a more specific \Magento\Framework\App\Action\HttpPostActionInterface. They have been around since Magento 2.3, so we shouldn't have any compatibility issues.

Reply:
This is done